src/deployment/set_admins.py

```diff
@@ -5,12 +5,20 @@
 
 import argparse
 import json
+from time import sleep
 from typing import List, Optional
 from uuid import UUID
 
 from azure.common.client_factory import get_client_from_cli_profile
 from azure.cosmosdb.table.tableservice import TableService
 from azure.mgmt.storage import StorageManagementClient
+import logging
+
+## Disable logging from storageclient. This modules display and error message
```

Comment:
```suggestion
## Disable logging from storageclient. This module displays an error message
```

Reply:
What other log messages do we lose by disabling this logger completely?

Is there a way we can avoid disabling it, for debugging deployments? E.g. maybe at higher verbosity levels, we should include logs from dependencies.

Reply:
I update the code to only disable the logging when we check for the table existence.